integration-test/src/test/java/org/apache/iotdb/relational/it/schema/IoTDBTableIT.java

```diff
@@ -761,6 +761,74 @@ public void testConcurrentAutoCreateAndDropColumn() throws Exception {
     }
   }
 
+  @Test
+  public void testTableObjectCheck() throws Exception {
+    try (final Connection connection =
+            EnvFactory.getEnv().getConnection(BaseEnv.TABLE_SQL_DIALECT);
+        final Statement statement = connection.createStatement()) {
+      statement.execute("create database db2");
+      statement.execute("use db2");
+      statement.execute("create table \".\" ()");
+
+      try {
+        statement.execute("alter table \".\" add column a object");
+        fail();
+      } catch (final SQLException e) {
+        Assert.assertEquals(
+            "701: When there are object fields, the tableName . shall not be '.', '..' or contain './', '.\\'",
+            e.getMessage());
+      }
```

Comment:
also need alter table statement whose table name is correct but column name is not correct.